src/libraries/System.Runtime.InteropServices.JavaScript/src/System/Runtime/InteropServices/JavaScript/JSSynchronizationContext.cs

```diff
@@ -35,31 +39,34 @@ internal sealed unsafe class JSSynchronizationContext : SynchronizationContext {
 
         private static JSSynchronizationContext? MainThreadSynchronizationContext;
         private readonly QueueType Queue;
-        private readonly Action _DataIsAvailable;
+        private static void* BackgroundJobHandlerPtr = (void*)(delegate* unmanaged[Cdecl]<void>)&BackgroundJobHandler;
```

Comment:
This isn't necessary.  `&SomeMethod` is basically a constant. Function pointers don't need to be cached in a static like this.